src/lib/models/events-group.ts

```diff
@@ -128,7 +156,12 @@ export class EventGroups {
   }
 
   add(event: CompactEvent): void {
-    if (isActivityScheduledEvent(event)) {
+    if (
+      isActivityScheduledEvent(event) ||
```

Comment:
Eventually we will probably want to break this out into something like `isGroupableEvent` or something.